### DISCUSSION TITLE
Use mode

### DISCUSSION THREAD

**Discussion by oymyakon** [wills106/homeassistant-solax-modbus#378] [General]
Hi guys,
My use mode was working until this morning. It just reverts back to the mode set up on the Solax X1.
All of the readings etc work i just cannot change the mode.
Any ideas?
My log is showing this error:-
Detected blocking call to sleep inside the event loop. This is causing stability issues. Please report issue to the custom integration author for solax_modbus doing blocking calls at custom_components/solax_modbus/__init__.py, line 318: return self._client.read_holding_registers(address, count, **kwargs)

**Comment by wills106** [comment]
Try pressing the Unlock button first.
Also if the Inverter is asleep you can't write to it.

**Reply by oymyakon** [reply]
I think that has worked thanks a lot.
Paul.
